ext/nnstreamer/tensor_source/tensor_src_grpc.c

```diff
@@ -290,6 +290,7 @@ _grpc_callback (void *obj, void *data)
   GST_BUFFER_PTS (buffer) = timestamp;
 
   item = g_new0 (GstDataQueueItem, 1);
+  g_assert (item != NULL);
```

Comment:
Q: Does any static analyzer report that they're defects?

Reply:
Yes, they did.
Though there are many cases that have no NULL checker after allocations, the recommendation is it.
It sounds quite weird, but they report only these cases as a defect. :sweat_smile:

Reply:
What about setting it as **False-Positive** instead of adding assertion?